packages/front-end/components/Layout/ProjectSelector.tsx

```diff
@@ -99,6 +153,14 @@ export default function ProjectSelector() {
                   avatarName={p.name}
                   display={p.name}
                   bold={p.id === project}
+                  badge={
+                    isDemoDatasourceProject({
+                      projectId: p.id,
+                      organizationId: orgId || "",
+                    })
+                      ? badgeProps
+                      : null
+                  }
```

Comment:
You shouldn't need a `isDemoDatasourceProject` check here.  `badgeProps` is null if it's not a demo project, so you can just always pass that in.

Reply:
This check is in the map, we need to check if `p.id` is the demo project so that it shows in the projects available to select, not if the current project that's selected is. 

What we want (current code):

<img width="235" alt="image" src="https://github.com/growthbook/growthbook/assets/113377031/83d85359-c45d-42b0-861e-b3b101589b0b">

What we'd get with your suggestion:

<img width="229" alt="image" src="https://github.com/growthbook/growthbook/assets/113377031/47ae6f43-5afb-4fb7-80b8-3d47e38b7981">